### ISSUE TITLE
Crash reporting with Firebase.

### ISSUE THREAD

**Issue by segunfamisa** [codepath/android_guides#179]
A guide on how to implement crash reporting with the new firebase:
https://hackmd.io/MYTgLAjADBAmYFoTIBwLAJgEZoIZjAFYFgA2XAUwDMQUNSBmXCIA

**Comment by rogerhu** [comment]
Nice!

Here are some of my suggestions:

1) I'd link to the Proguard guide ([[Configuring-Proguard]]) --
http://guides.codepath.com/android/Configuring-ProGuard
2) Is there a programmatic way to upload the Proguard mappings or does it
have to be done through the UI?
3) It feels like some of the setup flow is common -- wonder if we need a
common setup section just on the main Firebase guide page.

**Comment by segunfamisa** [comment]
Hi @rogerhu thanks for the suggestions. They're valid and very helpful.

1. I have updated the guide https://hackmd.io/MYTgLAjADBAmYFoTIBwLAJgEZoIZjAFYFgA2XAUwDMQUNSBmXCIA to reference the proguard guide.

2. I'm not sure if there's a programmatic way to upload the Proguard mappings. I'm only certain of the UI way.

3. Yes, you're correct, there's a similar setup flow, and as a result, I've modified this guide: https://hackmd.io/MYTgLAjADBAmYFoTIBwLAJgEZoIZjAFYFgA2XAUwDMQUNSBmXCIA to cover Firebase setup, and I've created another guide: https://hackmd.io/JwIwhmBmBswBwFoCmSDMAmBAWAjFsCIADFkskcFACbBXTTpVA=== to cover Crash reporting in firebase. 

Let me know what you think.

Cheers :)
